addons/website_event_questions/views/event_registration_answer_views.xml

```diff
@@ -50,6 +50,14 @@
     <record id="action_event_registration_report" model="ir.actions.act_window">
         <field name="name">Answer Breakdown</field>
         <field name="res_model">event.registration.answer</field>
-        <field name="view_mode">search,tree,graph,pivot</field>
+        <field name="view_mode">tree,graph,pivot</field>
+        <field name="search_view_id" ref="event_registration_answer_view_search"/>

```

Comment:
For future reviewer: I could not find any occurrence of an helper on a view_mode containing the search view, hence the change with search_view_id which seems to be the rule of thumb.